static/js/place/chart_block.tsx

```diff
@@ -70,6 +72,10 @@ class ChartBlock extends React.Component<ChartBlockPropType, unknown> {
     // Plot trend data in overview and topic page.
     if (!_.isEmpty(this.props.data.trend)) {
       const id = randDomId();
+      const rankingParam = new URLSearchParams(`h=${this.props.dcid}`);
+      this.props.data.denominator && rankingParam.set("pc", "1");
```

Comment:
this has no efffect?

Reply:
it adds pc=1 to rankingParam, only if this.props.data has a denominator. it was getting long with all the if statements, so i used shorthand -- i can unroll it tho